{"links": {"self": {"href": "data/repositories/osrf/subt/issues/136/comments/53194301.json"}, "html": {"href": "#!/osrf/subt/issues/136#comment-53194301"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/subt/issues/136.json"}}, "type": "issue", "id": 136, "repository": {"links": {"self": {"href": "data/repositories/osrf/subt.json"}, "html": {"href": "#!/osrf/subt"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3c95f6ad-c304-407c-b838-09597d836552}ts=2272898"}}, "type": "repository", "name": "subt", "full_name": "osrf/subt", "uuid": "{3c95f6ad-c304-407c-b838-09597d836552}"}, "title": "X3/X4 Not Responding to Velocity Commands in Local Cloudsim"}, "content": {"raw": "I was unable to reproduce this problem. Publishing on X3/cmd_vel or X4/cmd_vel works for me. Can you check if you have the latest docker image? Also, could you run `rqt_graph` and attach the image? It would be helpful to see if there any missing nodes.", "markup": "markdown", "html": "<p>I was unable to reproduce this problem. Publishing on X3/cmd_vel or X4/cmd_vel works for me. Can you check if you have the latest docker image? Also, could you run <code>rqt_graph</code> and attach the image? It would be helpful to see if there any missing nodes.</p>", "type": "rendered"}, "created_on": "2019-07-29T18:32:07.511887+00:00", "user": {"display_name": "Addisu Z. Taddese", "uuid": "{9d7fdc68-d270-4080-bf03-4df0d0dbf635}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B9d7fdc68-d270-4080-bf03-4df0d0dbf635%7D"}, "html": {"href": "https://bitbucket.org/%7B9d7fdc68-d270-4080-bf03-4df0d0dbf635%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:90bc87b0-2886-4377-a17c-3a6629a697ff/afbf9be5-1e4c-4810-89d4-e58831f34b11/128"}}, "nickname": "azeey", "type": "user", "account_id": "557058:90bc87b0-2886-4377-a17c-3a6629a697ff"}, "updated_on": null, "type": "issue_comment", "id": 53194301}